Bundle/BlogBundle/Repository/CategoryRepository.php

```diff
@@ -1,15 +1,16 @@
 <?php
 namespace Victoire\Bundle\BlogBundle\Repository;
 
-use Victoire\Bundle\BlogBundle\Entity\Blog;
 use Gedmo\Tree\Entity\Repository\NestedTreeRepository;
+use Victoire\Bundle\BlogBundle\Entity\Blog;
+use Victoire\Bundle\CoreBundle\Repository\ChainedRepositoryTrait;
 
 /**
  * The Category repository
  */
 class CategoryRepository extends NestedTreeRepository
 {
-    use \AppVentus\Awesome\ShortcutsBundle\Repository\AwesomeRepositoryTrait;
+    use ChainedRepositoryTrait;
```

Comment:
Is "Chained" the write class name ? It makes me think of a collection of services (like the filter_chain in Victoire FilterBundle). Maybe something like "StatefulRespoitoryTrait" ?